packages/root-repo/build.sh

```diff
@@ -1,26 +1,38 @@
-TERMUX_PKG_HOMEPAGE=https://github.com/termux/termux-root-packages
+TERMUX_PKG_HOMEPAGE=https://github.com/termux/termux-packages
 TERMUX_PKG_DESCRIPTION="Package repository containing programs for rooted devices"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="Henrik Grimler @Grimler91"
-TERMUX_PKG_VERSION=2.4
-TERMUX_PKG_REVISION=2
+TERMUX_PKG_VERSION=2.5
 TERMUX_PKG_AUTO_UPDATE=false
 TERMUX_PKG_DEPENDS="termux-keyring"
 TERMUX_PKG_SKIP_SRC_EXTRACT=true
 TERMUX_PKG_PLATFORM_INDEPENDENT=true
+TERMUX_PKG_CONFFILES="
+	etc/apt/sources.list.d/root.sources
+"
 
 termux_step_make_install() {
-	mkdir -p $TERMUX_PREFIX/etc/apt/sources.list.d
-	{
-		echo "# The root termux repository, with cloudflare cache"
-		echo "deb https://packages-cf.termux.dev/apt/termux-root/ root stable"
-		echo "# The root termux repository, without cloudflare cache"
-		echo "# deb https://packages.termux.dev/apt/termux-root/ root stable"
-	} > $TERMUX_PREFIX/etc/apt/sources.list.d/root.list
+	mkdir -p "$TERMUX_PREFIX/etc/apt/sources.list.d"
+	{ # write deb822 source entries for the root repo
+	printf '%s\n' \
+		"# The termux root repository, with cloudflare cache" \
+		"Types: deb" \
+		"URIs: https://packages-cf.termux.dev/apt/termux-root/" \
+		"Suites: stable" \
+		"Components: main" \
+		"Signed-By: $TERMUX_PREFIX/etc/apt/trusted.gpg.d/grimler.gpg, $TERMUX_PREFIX/etc/apt/trusted.gpg.d/termux-autobuilds.gpg" \
+		"" \
+		"# The termux root repository, without cloudflare cache" \
+		"#Types: deb" \
+		"#URIs: https://packages.termux.dev/apt/termux-root/" \
+		"#Suites: stable" \
+		"#Components: main" \
+		"#Signed-By: $TERMUX_PREFIX/etc/apt/trusted.gpg.d/grimler.gpg, $TERMUX_PREFIX/etc/apt/trusted.gpg.d/termux-autobuilds.gpg"
+	} > "$TERMUX_PREFIX/etc/apt/sources.list.d/root.sources"
 }
 
 termux_step_create_debscripts() {
-	[ "$TERMUX_PACKAGE_FORMAT" = "pacman" ] && return 0
+	[[ "$TERMUX_PACKAGE_FORMAT" == 'pacman' ]] && return 0
 	cat <<- EOF > ./postinst
```

Comment:
The `postinst` code is common for all `*-repo` packages, probably should be placed as a file in `apt` package and simply copied in `termux_step_create_debscripts` instead of writing them from scratch.

Reply:
`apt` itself doesn't use this `postinst`.